### PR TITLE
Follow new UA convention

### PR DESCRIPTION
**Pull request by JanPetr**

### CODE REVIEW

File: src/AlgoliaSearch/Version.php [hunk 1]
```diff
@@ -32,9 +32,10 @@ class Version
     const VALUE = '1.10.1';
 
     public static $custom_value = '';
+    public static $custom_value_prefix = '';
 
     public static function get()
     {
-        return self::VALUE.static::$custom_value;
+        return self::$custom_value_prefix . 'Algolia for PHP ('.self::VALUE.')' . static::$custom_value;
```
Comment (rayrutjes) [inline src/AlgoliaSearch/Version.php]: This breaks BC as we didn't have Algolia for PHP part of this before.

File: src/AlgoliaSearch/Version.php [hunk 2]
```diff
@@ -33,8 +33,32 @@ class Version
...
     public static $custom_value = '';
 
+    private static $prefixUserAgentSegments = '';
+    private static $suffixUserAgentSegments = '';
+
+    // Method untouched to keep backward compatibility
     public static function get()
     {
-        return self::VALUE.static::$custom_value;
+        return self::VALUE . static::$custom_value;
```
Comment (maxiloc) [inline src/AlgoliaSearch/Version.php]: Why can't we just do a concatenation ?

Comment (JanPetr) [inline src/AlgoliaSearch/Version.php]: ? We still do, there are just spaces for better readability.

Comment (maxiloc) [inline src/AlgoliaSearch/Version.php]: The thing I was saying is I think we can do everything from the get method and don't need the others

Comment (JanPetr) [inline src/AlgoliaSearch/Version.php]: - there is new UA convention, you can find it in API specifications. There should be parantheses around version numbers. It's impossible to do that in `get()` method with keeping with BC
- By the convention, the integration version should be before API client
- By convention `;` should be used as separator and version numbers should be in parantheses. It's always easier to keep this rules on one place (`Version` class) and use it's methods than keeping those rules in mind when dealing with all the integrations

Comment (maxiloc) [inline src/AlgoliaSearch/Version.php]: I see no issue in changing the get method. It don't see how it could impact existing users.

Comment (JanPetr) [inline src/AlgoliaSearch/Version.php]: Previously the method returned just version number. The probability is very low that somebody is using it but it's still a BC break. I wouldn't touch as there is no need to touch it.

Comment (maxiloc) [inline src/AlgoliaSearch/Version.php]: ok makes sense

**Pull request merged**